script/config/tls/master_template.conf

```diff
@@ -0,0 +1,11 @@
+port 0
+tls-port 46380
+tls-cert-file /etc/redis.crt
+tls-key-file /etc/redis.key
+tls-ca-cert-file /etc/ca-root-cert.pem
+tls-auth-clients optional
+tls-replication yes
+user default off
+user porta on >sup3rS3cre1! ~* &* +@all
+user apisonator on >secret#Passw0rd ~* &* +@all
```

Comment:
Should we mark these as non-sectets too?

Reply:
Well, these ones are obviously fake, but for some reason don't trigger an alert, so I think we are fine.

Reply:
Same for all other occurrences of this